src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -1375,29 +1375,30 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
         fn foo2() -> i32 { unimplemented!() }
     """)
 
-    fun `test trait method can have arg with Self type E0277`() = checkErrors("""
+    fun `test trait method without body can have arg with '?Sized' type E0277`() = checkErrors("""
+        #[lang = "sized"] trait Sized {}
         trait Foo {
             fn foo(x: Self);
+            fn bar(x: <error>Self</error>) {}
+            fn foobar() -> Self;
+            fn baz() -> <error>Self</error> {}
```

Comment:
```suggestion
            fn baz() -> <error>Self</error> { unimplemented!() }
```
Just to avoid unrelated error annotation here in future